system/services/admin/LoginService.cfc

```diff
@@ -732,10 +732,8 @@ component displayName="Admin login service" {
 	 * @autodoc
 	 *
 	 */
-	public void function disableTwoFactorAuthenticationForUser() {
-		var userId = getLoggedInUserId();
-
-		if ( userId.len() ) {
+	public void function disableTwoFactorAuthenticationForUser( required string userId=getLoggedInUserId() ) {
```

Comment:
If there is a default, do not make the argument required. so should be `disableTwoFactorAuthenticationForUser( string userId=getLoggedInUserId() ) {...`